service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderProgressionServiceTest.java

```diff
@@ -120,19 +192,35 @@ void shouldDoNothingWhenJudgeLeavesInSelfReview() {
                 .build());
     }
 
-    @SuppressWarnings("unchecked")
-    private CaseDetails getCaseDetails(CaseData caseData) {
-        Map<String, Object> data = mapper.convertValue(caseData, Map.class);
-        return CaseDetails.builder().data(data).build();
-    }
-
     private CaseData.CaseDataBuilder caseDataWithCmoToAction(ActionType type) {
         return CaseData.builder()
             .cmoToAction(CaseManagementOrder.builder()
                 .id(UUID)
+                .status(SEND_TO_JUDGE)
                 .action(OrderAction.builder()
                     .type(type)
                     .build())
                 .build());
     }
+
+    @SuppressWarnings("unchecked")
+    private CaseDetails getCaseDetails(CaseData caseData) {
+        Map<String, Object> data = mapper.convertValue(caseData, Map.class);
```

Comment:
If you use `new TypeReference<>(){}` here instead of `Map.class` you shouldn't need to suppress any unchecked casts